src/components/input-date-picker/input-date-picker.stories.ts

```diff
@@ -5,6 +5,41 @@ import readme from "./readme.md";
 import { html } from "../../../support/formatting";
 import { locales } from "../../utils/locale";
 
+const placements = [
```

Comment:
I think we can import these constants from the popper.ts file. https://github.com/Esri/calcite-components/blob/master/src/utils/popper.ts#L31